python/ambassador/envoy/v2/v2listener.py

```diff
@@ -897,6 +905,12 @@ def finalize(self) -> None:
 
             self.filter_chains.append(filter_chain)
 
+        if self.use_proxy_proto:
+            self.listener_filters.append({
+                'name': 'envoy.listener.proxy_protocol',
+                'config': {}
```

Comment:
Non-zero chance we'll have to fuss with `config`/`typed_config` when we update to 1.13.

Reply:
Yeah. Envoy's examples imply that `typed_config: {}` may be most correct, but something (`ambex`?) doesn't like that. I'll try a couple of other things but I'd vote for not holding it for that.